run.py

```diff
@@ -397,19 +458,11 @@ def run_state_level_forecast(
             max_date,
             output_dir,
         )
-        pool.apply_async(forecast_each_state, args=args)
+        pool.apply_async(
+            forecast_each_state,
+            args,
+            callback=_result_callback_wrapper(f"{state}, {country}")
+        )
 
     pool.close()
     pool.join()
-
-
-if __name__ == "__main__":
-    logging.basicConfig(level=logging.INFO)
-    with public_data_hash(os.getenv('COVID_DATA_PUBLIC_HASH', None)) as git_hash:
-        # @TODO: Record git hash in output data for reproducibility
-        # @TODO: Remove interventions override once support is in the Harvard model.
-        min_date = datetime.datetime(2020, 3, 7)
-        max_date = datetime.datetime(2020, 7, 6)
-        # build_county_summary()
-        run_county_level_forecast(min_date, max_date)
-        run_state_level_forecast(min_date, max_date)
```

Comment:
Er, why is all of this deleted?

Reply:
ah, i'll leave it for now actually.  Most of this logic for running the models now lives in `./run_model.py`